remote-backup/run.sh

```diff
@@ -101,10 +101,10 @@ function create-local-backup {
         fi
         bashio::log.info "Creating partial backup"
         bashio::log.debug "Including ${FOLDERS} and ${ADDONS}"
-        slug=$(ha backups new --raw-json --name="${name}" ${ADDONS} ${FOLDERS} | jq --raw-output '.data.slug')
+        slug=$(ha backups new --raw-json --name="${name}" ${ADDONS} ${FOLDERS} --password="${PASSWORD}" | jq --raw-output '.data.slug')
```

Comment:
Did you check whether this also works with an empty password?

Reply:
Yes, when an empty string is given it creates a backup without a password